### PR TITLE
Add family and friends

### PR DESCRIPTION
**Pull request by ghost**
Screen Shot 2022-08-13 at 9 46 37 PM
Screen Shot 2022-08-13 at 9 46 46 PM
Screen Shot 2022-08-13 at 9 46 55 PM
Screen Shot 2022-08-13 at 9 47 06 PM

### CODE REVIEW

File: content/relationships.html [hunk 1]
```diff
@@ -3,72 +3,86 @@ <h1 id="relationships">Family and Relationships</h1>
...
+    The SE cohort is a tight knit group.
+  </span>
+  ~56% of respondents said they had met or spoken with over 60% of the class. However, we should acknowledge that those who were more likely to fill out the class profile may have also been more likely to met others in the program.
+</p>
+<div class="half second">
+  <span class="graph-title">What percentage of people who started in or are in SE22 have you ever met / spoken to?</span>
+  <div id="se-people-met"></div>
 </div>
+
+<!-- TODO: Add quotes to this section. -->
```
Comment (ghost) [inline content/relationships.html:85]: Starting to add TODOs so we don't forget anything.

File: content/relationships.html [hunk 2]
```diff
@@ -3,72 +3,86 @@ <h1 id="relationships">Family and Relationships</h1>
...
 <div class="vb"></div>
 
 <h2>Friendships</h2>
-<div class="long_version">
 <p class="full">
-  On average, a softie made 4.36 new friends over co-op, and 3.05 over a school term.
-  They would lose on average 1.30 friends over co-op, compared to 0.55 over a school
-  term. Thus, the net gain in friends over co-op is about 3.1 friends, while the net
-  gain over a school term is 2.31 friends.
+  On average, a softie has 4.08 from high school, 1.36 friends from co-op, 1.48 from residence, and 1.49 from orientation. Furthermore, a softie has, on average, 5.17 friends in SE.
```
Comment (RobbieZhuang) [inline content/relationships.html]:
```suggestion
  On average, a softie has 4.08 friends from high school, 1.36 from co-op, 1.48 from residence, and 1.49 from orientation. Furthermore, a softie has, on average, 5.17 friends in SE.
```

File: content/relationships.html [hunk 3]
```diff
@@ -3,72 +3,86 @@ <h1 id="relationships">Family and Relationships</h1>
...
+<div class="half second">
+  <span class="graph-title">What percentage of your close friends are from SE?</span>
+  <div id="close-friend-se"></div>
 </div>
+
+<p class="half">
+  <span class="big">
+    The SE cohort is a tight knit group.
+  </span>
+  ~56% of respondents said they had met or spoken with over 60% of the class. However, we should acknowledge that those who were more likely to fill out the class profile may have also been more likely to met others in the program.
```
Comment (RobbieZhuang) [inline content/relationships.html]:
```suggestion
  ~56% of respondents said they had met or spoken with over 60% of the class. However, we should acknowledge that those who were more likely to fill out the class profile may have also been more likely to have met others in the program.
```

**Review by RobbieZhuang — APPROVED** [review]
For the Friendship Sources graph could we label the axis with something like the following:
x: Number of Friends
y: Number of Respondents

**Pull request merged**